model_analyzer/record/metrics_manager.py

```diff
@@ -28,7 +29,7 @@ class MetricsManager:
     This class handles the profiling
     categorization of metrics
     """
-    def __init__(self, config, metric_tags, server, result_manager):
+    def __init__(self, config, metric_tags, server, result_manager, gpus):
```

Comment:
Is this argument necessary still? We're passing in `config`. So could we use the `GPUDeviceFactory.get_analyzer_gpus` function call within this constructor?